subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/resolver/FindGradleSources.kt

```diff
@@ -36,6 +36,7 @@ import java.io.File
  */
 @DisableCachingByDefault(because = "Only filters the input artifact")
 abstract class FindGradleSources : TransformAction<TransformParameters.None> {
+    @get:PathSensitive(PathSensitivity.RELATIVE)
```

Comment:
Can we reuse the other `FindGradleSources` here somehow?

Reply:
I really don't know what the goal is for these two transforms called `FindGradleSources`, and why we have two different ones. Maybe @eskatos or @bamboo have some idea what is going on here.

If we want to unify those, I think this should go in a separate PR.

Reply:
I don't know/recall why we have two.
The Kotlin DSL one predates the IDE one that was introduced in https://github.com/gradle/gradle/pull/11785

Reply:
@rieske Do you know why we have two of those transforms?

Reply:
Most probably because I overlooked removing the one I added when I deleted those changes: https://github.com/gradle/gradle/pull/17683